package.json

```diff
@@ -410,9 +410,6 @@
       "eslint --rulesdir frontend/lint/eslint-rules --max-warnings 0 --fix",
       "prettier --write"
     ],
-    "e2e/test/scenarios/*/{*.(js|ts),!(helpers|shared)/*.(js|ts)}": [
-      "node e2e/validate-e2e-test-files.js"
-    ],
```

Comment:
We need to relax this requirement because we are writing tests in normal (non-`.cy.spec`) files too.